health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/services/SearchFileIds.js

```
@@ -0,0 +1,73 @@
+      useCache: false,
+      method: "POST",
+      userService: false,
+      body:body,
+    });
+    // debugger;
+    if (response?.ResourceDetails?.length === 0) {
+      throw new Error("File not found for given microplan");
+    }
+    let fileIds = []
+    let uuids = new Set(); //for making request to search user
+    for (const ob of response?.ResourceDetails) {
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)